ci/travis/linux/qt4/before_install.sh

```diff
@@ -20,5 +20,9 @@ pushd ${HOME}
 curl -L https://github.com/opengisch/osgeo4travis/archive/qt4bin.tar.gz | tar -xzC /home/travis --strip-components=1
 curl -L https://cmake.org/files/v3.5/cmake-3.5.0-Linux-x86_64.tar.gz | tar --strip-components=1 -zxC /home/travis/osgeo4travis
 
+#download OTB package 5.6 for processing test
+wget https://www.orfeo-toolbox.org/packages/OTB-5.6.0-Linux64.run -O /home/travis/OTB-5.6.0-Linux64.run && sh /home/travis/OTB-5.6.0-Linux64.run
+export OTB_APPLICATION_PATH=/home/travis/OTB-5.6.0-Linux64/lib/otb/applications
```

Comment:
I think you need to do the export in the script.sh file, if it's only in here it will not propagate.